test/gtest/ucp/test_ucp_rma.cc

```diff
@@ -182,7 +182,7 @@ class test_ucp_rma : public test_ucp_memheap {
     }
 
     bool disable_proto() {
-        return !m_ucp_config->ctx.proto_enable;
+        return get_variant_value() & DISABLE_PROTO;
```

Comment:
seems it is only used in `init` method now.
maybe it's better for check for `get_variant_value() & DISABLE_PROTO` there directly?

Reply:
@yosefe, do you think it's worth to cnosider?
seems such method is removed in other places

Reply:
done